src/service.js

```diff
@@ -37,7 +37,9 @@ class Service {
     const q = this.Model.find(query).lean(this.lean);
 
     // $select uses a specific find syntax, so it has to come first.
-    if (filters.$select && filters.$select.length) {
+    if (typeof filters.$select === 'string') {
```

Comment:
I don't think I can because of how the current check works on line 43, a string has a length right?(I just double checked in my console :) 

I could modify it, but I'd rather leave it alone....